core/domain/change_domain.py

```diff
@@ -74,6 +74,15 @@ def validate_cmd(cmd_name, valid_cmd_attribute_specs, actual_cmd_attributes):
     if error_msg_list:
         raise utils.ValidationError((', ').join(error_msg_list))
 
+    deprecated_values = valid_cmd_attribute_specs.get('deprecated_values')
+    if deprecated_values:
+        for attribute_name, attribute_values in deprecated_values.items():
+            actual_value = actual_cmd_attributes[attribute_name]
+            if actual_value in attribute_values:
+                raise utils.DeprecatedError(
+                    'Value for %s in cmd %s: %s is deprecated' % (
+                        attribute_name, cmd_name, actual_value))
```

Comment:
Ditto as above.

Reply:
```suggestion
    deprecated_values = valid_cmd_attribute_specs.get('deprecated_values', {})
    for attribute_name, attribute_values in deprecated_values.items():
        actual_value = actual_cmd_attributes[attribute_name]
        if actual_value in attribute_values:
            raise utils.DeprecatedError(
                'Value for %s in cmd %s: %s is deprecated' % (
                    attribute_name, cmd_name, actual_value))
```

Reply:
Done.

Reply:
Reverting this, since this if condition is necessary otherwise deprecated_values.items is not defined.

Reply:
That is why I used `get('deprecated_values', {})` 
![image](https://user-images.githubusercontent.com/10142938/122039868-691b6d80-cdd7-11eb-88c7-a7c494498cc5.png)

Reply:
Done.